### DISCUSSION TITLE
Release v0.5.0

### DISCUSSION THREAD

**Discussion by martialblog** [NETWAYS/go-check#77] [Ideas] [answered]
This discussion will serve as a collecting point for everything regarding the Release v0.5.0

**Comment by RincewindsHat** [comment]
I need to generalize the threshold values. The members which are still `float64` should be exchanged for a _numeric_ `interface`

**Comment by martialblog** [comment]
@RincewindsHat and I think the release should focus on cleanup

- Remove some already deprecated functions
- Update docs and README
- Remove the benchmark package since its usefulness in the separate is unclear
- Remove the http/mock package since not all check plugins use http and it should be a separate library

**Comment by martialblog** [comment, marked as answer]
RC1 is out https://github.com/NETWAYS/go-check/releases/tag/v0.5.0-rc1
